templates/package.json.ts

```diff
@@ -70,9 +70,12 @@ const templateFunction: TemplateFunction = async answers => {
 			"react-dom",
 			"@types/react",
 			"@types/react-dom",
-			// UI library support in TS:
-			"@types/jquery",
-			"@types/materialize-css",
+			// ioBroker
+			"@iobroker/adapter-react",
+			// UI library
+			"react-icons",
```

Comment:
Why do we need two icon libraries? And does `react-icons` include its on definitions?

Reply:
Honestly: no clue. Took this from Bluefox's example.